ui2/src/websocket/queryStream.ts

```diff
@@ -52,8 +52,8 @@ export const getPublicToken = async (publicParty: string): Promise<string | unde
   return publicToken;
 };
 
-const QueryStreamProvider = (props: PropsWithChildren<any>) => {
-  const { children } = props;
+const QueryStreamProvider = (props: PropsWithChildren<any> & { defaultPartyToken?: string }) => {
+  const { children, defaultPartyToken } = props;
```

Comment:
this will be used in Quick Setup to avoid "logging in" a.k.a. setting global credentials .. but still be able to query as a party..

Reply:
Ah, good idea!